tap_mixpanel/streams.py

```diff
@@ -297,6 +297,13 @@ def sync(self, state: dict, catalog, config: dict):
         days_interval = int(config.get("date_window_size", "30"))
         attribution_window = int(config.get("attribution_window", "5"))
 
+        #Update url if eu_residency_server is selected
+        if config.get('eu_residency_server'):
```

Comment:
@prijendev Please update this condition as per https://github.com/singer-io/tap-xero/blob/master/tap_xero/streams.py#L122

Reply:
done